kubetest2-gce/deployer/build.go

```diff
@@ -39,8 +40,16 @@ func (d *deployer) Build() error {
 		if err != nil {
 			return err
 		}
+
 		// append the kubetest2 run id
-		version += "+" + d.commonOptions.RunID()
+		// avoid double + in the version
+		// so they are valid docker tags
+		if strings.Contains(version, "+") {
```

Comment:
We have any tests for these?

Reply:
we have them over on the gke deployer side, but good point we shouid have some here as well.

we lack a lot of unit tests in general majorly because most of the things are cmd.exec /shrug 
I'll be working on improving that story